tests/test_events.py

```diff
@@ -287,7 +287,12 @@ def test_order_started(events_object, client, base_ddo_url):
         ],
     )
     pk = keys.PrivateKey(provider_wallet.key)
-    signed = keys.ecdsa_sign(message_hash=message, private_key=pk)
+    prefix = "\x19Ethereum Signed Message:\n32"
+    messageHash = Web3.solidityKeccak(
```

Comment:
Consider changing to `signable_hash`
Consider changing `message` on line 280 to `message_hash`
See similar comment for reasoning why: https://github.com/oceanprotocol/provider/pull/317#discussion_r777718367